lib/nexpose/vuln_exception.rb

```diff
@@ -3,31 +3,69 @@ module Nexpose
   class Connection
     include XMLUtils
 
-    # Retrieve vulnerability exceptions.
+    # Retrieve all active vulnerability exceptions.
     #
     # @param [String] status Filter exceptions by the current status.
     #   @see Nexpose::VulnException::Status
-    # @param [String] duration A time interval in the format "PnYnMnDTnHnMnS".
```

Comment:
Should we leave this param in and filter on the returned results? We're doing that for the status anyway right?